pytezos/operation/types.py

```diff
@@ -0,0 +1,16 @@
+from typing import TypedDict, Any
+
+
+class OperationGroupContent(TypedDict, total=False):
```

Comment:
Check out content layouts at https://github.com/baking-bad/pytezos/blob/master/pytezos/operation/content.py

Reply:
Probably better to introduce type variants instead of combining all fields in a single class, i.e.:

```
class TransactionContent(OperationContent):
    pass
    
class OriginationContent(OperationContent):
    pass
```

Related to #157